src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -648,12 +651,127 @@ public Chat getChat() {
+    if (!comments.contains("automated_host")) {
+      System.out.println(
+          "Invalid argument: " + GameRunner.LOBBY_GAME_COMMENTS + " must contain the string \"automated_host\".");
+      printUsage = true;
+    }
+
+    final String email = System.getProperty(GameRunner.LOBBY_GAME_SUPPORT_EMAIL, "");
+    if (email.length() < 3 || !Util.isMailValid(email)) {
+      System.out.println(
+          "Invalid argument: " + GameRunner.LOBBY_GAME_SUPPORT_EMAIL + " must contain a valid email address.");
+      printUsage = true;
+    }
+
+    final String reconnection = System.getProperty(GameRunner.LOBBY_GAME_RECONNECTION,
+        "" + GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT);
+    try {
+      final int reconnect = Integer.parseInt(reconnection);
+      if (reconnect < GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM) {
+        System.out.println("Invalid argument: " + GameRunner.LOBBY_GAME_RECONNECTION
+            + " must be an integer equal to or greater than " + GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM
+            + " seconds, and should normally be either " + GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT
+            + " or " + (2 * GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT) + " seconds.");
+        printUsage = true;
+      }
+    } catch (final NumberFormatException e) {
+      System.out.println("Invalid argument: " + GameRunner.LOBBY_GAME_RECONNECTION
+          + " must be an integer equal to or greater than " + GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM
+          + " seconds, and should normally be either " + GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT + " or "
+          + (2 * GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT) + " seconds.");
+      printUsage = true;
+    }
+    // no passwords allowed for bots
+    // take any actions or commit to preferences
+    final String clientWait = System.getProperty(GameRunner.TRIPLEA_SERVER_START_GAME_SYNC_WAIT_TIME, "");
+    final String observerWait = System.getProperty(GameRunner.TRIPLEA_SERVER_OBSERVER_JOIN_WAIT_TIME, "");
+    if (clientWait.length() > 0) {
+      try {
+        final int wait = Integer.parseInt(clientWait);
+        GameRunner.setServerStartGameSyncWaitTime(wait);
+      } catch (final NumberFormatException e) {
+        System.out.println(
+            "Invalid argument: " + GameRunner.TRIPLEA_SERVER_START_GAME_SYNC_WAIT_TIME + " must be an integer.");
+        printUsage = true;
+      }
+    }
+    if (observerWait.length() > 0) {
+      try {
+        final int wait = Integer.parseInt(observerWait);
+        GameRunner.setServerObserverJoinWaitTime(wait);
+      } catch (final NumberFormatException e) {
+        System.out.println(
+            "Invalid argument: " + GameRunner.TRIPLEA_SERVER_START_GAME_SYNC_WAIT_TIME + " must be an integer.");
+        printUsage = true;
+      }
+    }
+
+    if (printUsage) {
+      usage();
+      System.exit(-1);
+    }
```

Comment:
I understand this is inherited code from the old `GameRunner#handleCommandLineArgs()`, and you may intend to do this in a future refactoring, but pointing it out just in case...  It might be cleaner if this method returned a `boolean` just like `ArgParser#handleCommandLineArgs()` to avoid all the `printUsage` usage and avoid the call to `System#exit()`; the `usage()` call could be moved to `main()`.